src/debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -328,6 +334,17 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
         return false;
     }
 
+    private async inferUserSecrets(ssl: boolean, projectFile: string): Promise<boolean> {
+        if (ssl) {
+            return true;
+        }
+
+        if (this.fsProvider.fileExists(projectFile)) {
```

Comment:
`inferUserSecrets` is async. Does the `fileExists` check have an async version too, that we can use here?

Reply:
fileExists in fact is async, and this if statement is broken. It would always think it's true, since the promise would always be defined (either true or false). Good catch!